### ISSUE TITLE
how to verify RS256 signing algorithm?

### ISSUE THREAD

**Issue by OscarZhou** [dgrijalva/jwt-go#232]
I met a problem when I try to verify the JWT returned from Auth0. In Auth0, I use the client credentials to ask the token, as the result, the Auth0 returned me a JWT. In this case, I do not know how to verify it.  

I met an another problem about Auth0 that the front-end send me (back-end) a token to request an API, but I have to transmit this token to Auth0 to verify it again. So far, I think I need to form a JWT to Auth0, but I do not know how to parse the previous JWT (authorize me, which is the above question). Who knows that?

**Comment by karlbateman** [comment]
You can and should verify the token on the frontend. This library exposes validation functionality but is lacking an example for Parsing/Validating RS256 tokens.

**Ref** [Parse HMAC Example](https://godoc.org/github.com/dgrijalva/jwt-go#example-Parse--Hmac)

**Comment by OscarZhou** [comment]
Hi Karl, Thank you so much. I have already figured this problem out. At that time, I did not understand the authentication workflow in our project. I did the back-end part, so I can not do it on the front-end. The library I used are  

	jose "gopkg.in/square/go-jose.v2"
	"gopkg.in/square/go-jose.v2/jwt"

, which save me a lot of time. 
I found it from [here](https://auth0.com/blog/authentication-in-golang/), which is amazing. 
Anyway, thank you so much bro.
